ceph.spec.in

```diff
@@ -262,7 +262,11 @@ BuildRequires:	socat
 %if 0%{with zbd}
 BuildRequires:  libzbd-devel
 %endif
+%if 0%{?suse_version}
+BuildRequires:  libthrift-devel >= 0.13.0
```

Comment:
see https://pkgs.org/search/?q=libthrift-devel, so only openSUSE Tumbleweed has libthrift-devel > 0.13.0 ([libthrift-devel-0.15.0](https://opensuse.pkgs.org/tumbleweed/opensuse-oss-aarch64/libthrift-devel-0.15.0-1.1.aarch64.rpm.html) at the time of writing), but openSUSE Leap does not. given the connection between openSUSE Leap and SLE, i don't think SLE ships with libthrift-devel >= 0.13.0. shall we opt out of libthrift-devel on these two distros. as "%{suse_versoin} is defined to nonzero on them. by making `libthrift-devel >= 0.13.0` a `BuildRequires` would break the build on them as long as we still build on them.

@tserong @smithfarm what do you think?

Reply:
openSUSE Leap 15.2 (and thus SLE 15 SP2) ships Octopus, and Leap 15.3 and 15.4 (and thus SLE 15 SP3 and SP4) ship Pacific, so unless thrift gets backported to those earlier Ceph releases, Leap and SLE should be OK.